tests/phpunit/elementor/app/import-export/processes/test-import.php

```diff
@@ -475,6 +475,49 @@ public function test_get_manifest__kit_library() {
 		}
 	}
 
+	public function test_init_import_session__kit_thumbnail() {
+		// Arrange
+		$import_settings = [
+			'referrer' => 'kit-library',
+		];
+		$module = new Module();
```

Comment:
why do you run the whole module? can't you do it like the other tests?
something like: `( new Import() )->run()`

Reply:
The `import_kit` method wraps the `run` and makes all the neccessary arrangments that need to be made before calling `run` like ensuring writing permissions and registering default runners which we would need to do anyway if we were to call run directly in the test

Reply:
I know it wraps the import, but how is this necessary in this test?
every other test in this file doesn't use the module
The behavior of the module should be tested inside the module test, not inside this unit test of the import process.

It should also simplify the tests (e.g. you'll be able to remove this: `get_option( Module::OPTION_KEY_ELEMENTOR_IMPORT_SESSIONS )`, which is an implementation detail of the module)